### ISSUE TITLE
Domain added multiple times to groups

### ISSUE THREAD

**Issue by albertodonato** [CanonicalLtd/candid#125]
I have candid configured with two `static` IDP backends.
It seems that candid is saving the group with domain suffix after returning it.
This is what i see:

```
$ bhttp -C http://crbs:8081/v1/u/user1@foo.com
Opening an authorization web page in your browser.
If it does not open, please open this URL:
http://crbs:8081/login?did=80e7cea7d20f8bf25e5ba43e45c06b8cc87979e48a18399df0c36e9bc1c7e817
{
        username: "user1@foo.com"
        external_id: "static1:user1@foo.com"
        fullname: "User One"
        email: "user1@foo.com"
        gravatar_id: "3a170e6170c3977c988b31517c22c9da"
        idpgroups: [
                "group1@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com"                                                                                                                             
                "group3@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com"                                                                                                                             
        ]
        public_keys: []
        ssh_keys: null
        last_login: "2019-02-25T10:38:16.633072536Z"
        last_discharge: "2019-02-25T10:38:16.633860409Z"
}


$ bhttp -C http://crbs:8081/v1/u/user1@foo.com/groups
Opening an authorization web page in your browser.
If it does not open, please open this URL:
http://crbs:8081/login?did=b3fbb2b67bb9ff810218ea3c313a415a51f19f7f63aa634ae93ce66c5f9c56e4
[
        "group1@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com"                                                                                                                             
        "group3@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com@foo.com"                                                                                                                             
]
```

**Comment by mhilton** [comment]
This should have been fixed in #126

**Comment by albertodonato** [comment]
Indeed, thanks for the fix
